pylint/message/message.py

```diff
@@ -25,6 +25,8 @@
         "obj",
         "line",
         "column",
+        "end_lineno",
+        "end_col_offset",
```

Comment:
_Not a full review yet._

I was wondering, should we rename those to `end_line` and `end_column` respectively? That would match the existing `line` and `column`. Relevant as these are the arguments to `--msg-template`.

Reply:
I opted to follow what `ast` uses, but yeah this makes sense. I'll change!